test/services/caServiceTest.js

```diff
@@ -0,0 +1,74 @@
+const createCaService = require('../../server/services/caService')
+
+describe('caService', () => {
+  let roService
+  let caService
+  const lduActiveClient = { isLduPresent: () => {} }
+
+  describe('Responsible Officer is allocated and Ldu determined', () => {
+    const responsibleOfficerOrError = {
```

Comment:
maybe just call this `responsibleOfficer`? as it always is in this set of tests?

Reply:
done